src/shared/components/results/ResultsView.tsx

```diff
@@ -113,6 +126,21 @@ const ResultsView: FC<ResultsTableProps> = ({
     prevViewMode.current = viewMode;
   });
 
+  // redirect to entry directly when only 1 result and query marked as "direct"
```

Comment:
Aaaaannnnd that makes my other comment irrelevant 🤣

Reply:
😅 just matching current behaviour here, but we need to discuss that with UX I think, for the reasons raised in my reply to your other comment